src/client/components/PropertyListItem/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Card, Image, Button, CardContent, Grid, CardMeta, CardDescription, Container, Icon, Header, Label } from 'semantic-ui-react';
+import './index.scss';
+
+export default class PropertyListItem extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
All properties should be in props and not in state. Local state is needed only in rare cases, when you have some transient data.